protocols/yungapedao/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "YungApeDao",
```

Comment:
please make it all smallcase, this parameter is not used for display, we use it to retrieve protocols